docs/commands/completion.md

```diff
@@ -14,13 +14,15 @@ The output is a script for you to copy and paste, and add to the configuration f
 
```

Comment:
Put before the command, "Use the following command to generate the autocompletion script that can be added to your `.bashrc` or `.bash_profile`: